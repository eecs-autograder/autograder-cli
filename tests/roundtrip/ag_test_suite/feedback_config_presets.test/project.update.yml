project:
  name: Test Project 0bea2cd6a042419cb22a79650eef10c8
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
  test_suites:
  - name: Suite 1
    setup_cmd: echo "WAAAAA"
    normal_fdbk_config: public
    ultimate_submission_fdbk_config: public
    past_limit_submission_fdbk_config: pass/fail
    staff_viewer_fdbk_config: private
