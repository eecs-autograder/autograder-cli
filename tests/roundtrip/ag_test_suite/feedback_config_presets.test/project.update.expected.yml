project:
  name: Test Project 0bea2cd6a042419cb22a79650eef10c8
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings: {}
  test_suites:
  - name: Suite 1
    setup_cmd: echo "WAAAAA"
    past_limit_submission_fdbk_config: pass/fail
    staff_viewer_fdbk_config: private
